libkpod/hooks.go

```diff
@@ -0,0 +1,90 @@
+package libkpod
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// DefaultHooksDirPath Default directory containing hooks config files
+	DefaultHooksDirPath = "/usr/share/containers/oci/hooks.d"
+	// OverrideHooksDirPath Directory where admin can override the default configuration
+	OverrideHooksDirPath = "/etc/containers/oci/hooks.d"
```

Comment:
We provide a default override directory, but we don't let the user specify one in the configuration or via CLI flag. Is this worth including?

Reply:
We should be hiding the option to overide, this is just for the test suite.  I don't see the user actually modifying this option. 

The way I was thinking this would work, is by default we look for hooks definitions in 
DefaultHooksDirPath and OverrideHooksDirPath, but if the test tool uses the `hidden` --hooks-dir-path we only look in that specified path.